src/node/cli.ts

```diff
@@ -65,7 +45,6 @@ export interface Args
   json?: boolean
   log?: LogLevel
   open?: boolean
-  port?: number
```

Comment:
❌ is this a breaking change?

Reply:
This was removed since any server function we’d pass the args into won’t accept a `string | number` union, so instead the only additions we add are code-server specific

Reply:
Ahh okay, got it! Thanks for clarifying